packages/manager/modules/pci/src/projects/project/workflow/index.scss

```diff
@@ -1,9 +1,9 @@
 ovh-manager-kubernetes-list {
-  @import '~bootstrap4/scss/_functions';
-  @import '~bootstrap4/scss/_variables';
-  @import '~bootstrap4/scss/_mixins';
-  @import '~bootstrap4/scss/_utilities';
-  @import '~bootstrap4/scss/_grid';
+  @import 'bootstrap4/scss/_functions';
```

Comment:
It looks like we are importing bootstrap4 a lot, maybe we can import it at a parent level (out of scope, just a thought)

Reply:
Yes we might have something to do here, i also found that scoping bootstrap might look like it's a best practice but in reality it makes the css dist a LOTS heavier : each bootstrap's class definition is duplicated for the new scope.